src/coefficients/stdfunction_coefficient.hpp

```diff
@@ -148,18 +151,20 @@ class TransformedVectorCoefficient : public mfem::VectorCoefficient
      \param[in] func A function that takes in two input vectors, and returns the output as the third argument.
   */
 
-  TransformedVectorCoefficient(mfem::VectorCoefficient *v1, mfem::VectorCoefficient *v2,
-                               std::function <void (mfem::Vector &, mfem::Vector &, mfem::Vector &)>  func);
-  virtual void Eval (mfem::Vector & V, mfem::ElementTransformation & T, const mfem::IntegrationPoint & ip );
+  TransformedVectorCoefficient(
+    mfem::VectorCoefficient *v1,
+    mfem::VectorCoefficient *v2,
+    std::function<void(mfem::Vector &, mfem::Vector &, mfem::Vector &)> func);
+  virtual void Eval(mfem::Vector &V,
+                    mfem::ElementTransformation &T,
+                    const mfem::IntegrationPoint &ip);
 
-private:
+  private:
   mfem::VectorCoefficient *m_v1;
```

Comment:
private should not be at the same level as the member's.. private/public is usually not indented.